packages/web-maps/src/components/MapView.tsx

```diff
@@ -95,6 +95,9 @@ class MapView extends React.Component<MapViewProps, State> {
       scrollEnabled = true,
       mapType = "standard",
       style,
+      data,
```

Comment:
Should we give this a different name, like `points` or something? Just thinking that `data` isn't very descriptive

Reply:
Good call... how about `markersData`, to clarify that it's the data for the `MapMarker`s?

Reply:
I think for `Swiper`, `data` is probably ok unless there's another name that comes to mind for you